assembl/static2/js/app/utils/translations.js

```diff
@@ -1368,6 +1369,7 @@ const Translations = {
       sharePost: "Share this message",
       shareSynthesis: "Share this synthesis",
       copyLink: "Copy the link to the clipboard",
+      shareMail: "Share link via email",
```

Comment:
"Share the* link via e-mail", just to be consistent with the `copyLink` grammar above.